document/alert/alert.html

```diff
@@ -41,6 +41,7 @@ <h1 class="head-lg margin-xs">Components</h1>
         <a class="d-block btn-link" href="/document/button/button.html"
           >Button</a
         >
+        <a class="d-block btn-link" href="/document/card/card.html">Card</a>
```

Comment:
what is d-block class?

Reply:
like d-flex == display : flex then d-block == display: block